app/src/views/Home.vue

```diff
@@ -2,19 +2,34 @@
   <div class="relative h-full w-full items-center justify-center overflow-hidden">
     <AppBar v-model:search-input="searchInput" />
 
-    <DetailsPopup :is-open="!!selectedMarker" size="3/4" @close="selectedMarker = undefined">
+    <DetailsPopup
+      :is-open="!!selectedMarker"
+      :disable-resize="liteMode"
+      :size="liteMode ? '1' : '3/4'"
+      @close="selectedMarker = undefined"
+    >
       <MarkerPopup v-if="selectedMarker" :marker="selectedMarker" />
     </DetailsPopup>
 
-    <DetailsPopup :is-open="$route.name === 'search'" size="1/2" @close="$router.replace({ name: 'home' })">
+    <DetailsPopup
+      :is-open="$route.name === 'search'"
+      :disable-resize="liteMode"
+      :size="liteMode ? '1' : '1/2'"
+      @close="$router.replace({ name: 'home' })"
+    >
       <SearchPopup v-model:search-input="searchInput" />
     </DetailsPopup>
 
-    <DetailsPopup :is-open="$route.name === 'favorites'" size="1/2" @close="$router.replace({ name: 'home' })">
+    <DetailsPopup
+      :is-open="$route.name === 'favorites'"
+      :disable-resize="liteMode"
+      :size="liteMode ? '1' : '1/2'"
+      @close="$router.replace({ name: 'home' })"
+    >
       <FavoritesPopup />
     </DetailsPopup>
 
-    <Map :selected-marker="selectedMarker" @marker-click="selectedMarker = $event" />
+    <Map v-if="!liteMode" :selected-marker="selectedMarker" @marker-click="selectedMarker = $event" />
```

Comment:
@anbraten Should we lazy load the Map?

Reply:
@lukashass How could I do that?

Reply:
Did that by using `defineAsyncComponent`.